frontend/public/components/nav.jsx

```diff
@@ -233,6 +234,27 @@ const NavSection = connect(navSectionStateToProps)(
 
 const Sep = () => <div className="navigation-container__section__separator" />;
 
+
+const UserNavSectionWrapper = connectToFlags(FLAGS.AUTH_ENABLED, FLAGS.OPENSHIFT)(({flags, closeMenu}) => {
+  if (!flags[FLAGS.AUTH_ENABLED]) {
+    return null;
+  }
+
+  const logout = e => {
+    e.preventDefault();
+    authSvc.logout();
+  };
+
+  if (flags[FLAGS.OPENSHIFT]) {
+    return <NavSection text="Logout" icon="fa-user" klass="visible-xs-block" onClick={() => authSvc.logout()} />;
+  }
+
+  return <NavSection text="User" icon="fa-user" klass="visible-xs-block">
+    <HrefLink href="/settings/profile" name="My Account" onClick={closeMenu} key="myAccount" />
+    <HrefLink href="#" name="Logout" onClick={e => logout(e)} key="logout" />
```

Comment:
You've got two arrow function here since you already use an arrow function when declaring logout above. So you should be able to just say `onClick={logout}`.

It's generally recommended to bind to this in the constructor, though, instead of using arrow functions. A little about this in:

https://reactjs.org/docs/handling-events.html